src/components/modules/dashboard/offer/ui/dialogs/LoanRequestDetailDialog.tsx

```
@@ -0,0 +1,147 @@
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+} from "@/components/ui/dialog";
+import { useFormatUtils } from "@/utils/hook/format.hook";
+import { useCopyUtils } from "@/utils/hook/copy.hook";
+import { Copy, Check } from "lucide-react";
+
+interface LoanRequestDetailDialogProps {
+  isDialogOpen: boolean;
+  setIsDialogOpen: (value: boolean) => void;
+  selectedRequest?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)